modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -7,8 +7,23 @@
 
 // DUPE with SilverDialog. Cleaning up.
 import {
-  AddEventsBehaviour, AlloyEvents, AlloyTriggers, Behaviour, Composing, Focusing, GuiFactory, Keying, Memento, NativeEvents, Receiving, Reflecting,
-  SimpleSpec, SystemEvents
+  AddEventsBehaviour,
```

Comment:
Another import that should wrap, instead of chop.

- [ ] Fix formatting

Reply:
I think it's fixed, please check